docs/source/markdown/options/network.md

```diff
@@ -15,17 +15,23 @@ Valid _mode_ values are:
   - **mac=MAC**: Specify a static mac address for this container.
   - **interface_name**: Specify a name for the created network interface inside the container.
 
-  For example, to set a static ipv4 address and a static mac address, use `--network bridge:ip=10.88.0.10,mac=44:33:22:11:00:99`.
-
-- \<network name or ID\>[:OPTIONS,...]: Connect to a user-defined network; this is the network name or ID from a network created by **[podman network create](podman-network-create.1.md)**. Using the network name implies the bridge network mode. It is possible to specify the same options described under the bridge mode above. Use the **--network** option multiple times to specify additional networks.
+    For example, to set a static ipv4 address and a static mac address, use `--network bridge:ip=10.88.0.10,mac=44:33:22:11:00:99`.
```

Comment:
I don't think this looks right, it intends this to the same level as the interface_name but this clearly should be on the level from bridge which it is right now.

Reply:
Yea, it looks wrong in the text but the generated output is correct. Markdown takes 4 spaces as a unit of indentation.

Reply:
What generated output, html or man page?
It should not be on the nested list level as it applies to the bridge not interface_name so I think this change is incorrect

Reply:
I'm looking at the generated manpage and there it's correct, the example is on the same level as `bridge[...]`, not `interface_name`. However, in the html it's indeed incorrect. Not sure if this can be resolved.

Reply:
yeah I see it now, I was only looking at the markdown/html view.